src/models/Country.php

```diff
@@ -87,31 +92,28 @@ public function getTransform(): string {
       $country_id,
     );
 
-    await self::genDeleteAllMemcacheKeys();
+    self::invalidateMCRecords(); // Invalidate Memcached Country data.
   }
 
   private static async function genAll(
     string $sql,
-    string $mcKey,
   ): Awaitable<array<Country>> {
-    $mc = self::getMc();
-    $mc_result = $mc->get($mcKey);
+    $db = await self::genDb();
+    $all_countries = Map {};
+    /* HH_IGNORE_ERROR[4110] */
```

Comment:
This HH_IGNORE_ERROR was already in the code, from commit d5846c5. However, it appears to be related to the _$sql_ parameter being passed to _$db->queryf()_.  The _queryf()_ method is expecting a type ofHH\FormatString, but is being provided a standard string here.

With that said, none of the calls to Country::genAll() are providing any conditions or inputs into the queries.  Thus there is no need for a formatted string, and _$db->query()_ can be used instead of _$db->queryf()_.

This will be updated in the next commit and will remove the need for both of the HH_IGNORE_ERROR usages in this case.